tests/object_file_system_test.php

```diff
@@ -563,35 +562,41 @@ public function test_presigned_url_should_redirect_method_return_true_regardless
         $this->filesystem = new test_file_system();
         $externalclient = $this->filesystem->get_external_client();
 
-        if ($externalclient->support_presigned_urls()) {
-            $this->set_externalclient_config('enablepresignedurls', '1');
-            $this->set_externalclient_config('presignedminfilesize', 0);
-            $object = $this->create_local_object();
-            $this->assertTrue($this->filesystem->presigned_url_should_redirect($object->contenthash));
+        if (!$externalclient->support_presigned_urls()) {
+            $this->markTestSkipped('Pre-signed URLs not supported for given storage.');
         }
+
+        $this->set_externalclient_config('enablepresignedurls', '1');
+        $this->set_externalclient_config('presignedminfilesize', 0);
```

Comment:
Can you please also test scenarios when presignedminfilesize and enablepresignedurls are set to null, empty string, '0' as a string?